### PR TITLE
add github actions

### PR DESCRIPTION
**Pull request by tobowers**
This adds a github action for CI and does a tiny bit of logging cleanup

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - run: cd wasmtupelo && docker-compose up -d
```
Comment (brandonwestcott) [inline .github/workflows/test.yml]: I'm not sure if this is possible, but sometimes I find it helpful to put these longer series of commands into a `.sh` file - i personally think it makes it easier to manage and you get the syntax highlighting. This is still relatively small though and may not be possible

File: src/crypto.spec.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ describe('EcdsaKeys', ()=> {
         const key = await EcdsaKey.generate()
         expect(key.publicKey).to.have.length(65)
         expect(key.privateKey).to.have.length(32)
-    })
+    }).timeout(10000) // this is set to 10s because it's the first test that runs in CI
```
Comment (brandonwestcott) [inline src/crypto.spec.ts]: 👍 comment

**Comment by tobowers** [comment]
@brandonwestcott I like the way the logs get separated out in the UI with individual steps, but I agree multiline runs are pretty annoying to edit in the yaml... I moved the long ones out to separate scripts.

**Comment by tobowers** [comment]
I also added running the examples.

**Pull request merged**